daemon/container_operations_windows.go

```diff
@@ -58,3 +43,7 @@ func isLinkable(child *container.Container) bool {
 func enableIPOnPredefinedNetwork() bool {
 	return true
 }
+
+func (daemon *Daemon) supportsHotAdd() bool {
```

Comment:
I think I should probably make it supportsEndpointHotAdd. 
Having it as a compile time constant might be a good idea but I am not sure